Tests/Transformers/MysqlToRequestTest.php

```diff
@@ -226,7 +240,32 @@ public function updateAll() {
             'payload'     => json_encode(['name' => 'myValue'])
         ]);
 
-        $this->assertEquals($expected, $this->mysqlToRequest->transform($query));
+        $this->assertEquals($expected, $this->createMysqlToRequest()->transform($query));
+    }
+
+    /**
+     * @test
+     * @group  unit
+     * @covers ::__construct
+     * @covers ::transform
+     * @covers ::<private>
+     */
+    public function updatePatch() {
```

Comment:
This additional test has triggered phpmd to warn of >10 public methods in a class. I was unable to solve that.